docs/src/components/ComponentDoc/PerfChart/PerfChart.tsx

```diff
@@ -0,0 +1,92 @@
+import * as React from 'react'
+import * as _ from 'lodash'
+import { FlexibleXYPlot, XAxis, YAxis, HorizontalGridLines, LineSeries, Crosshair } from 'react-vis'
+
+const ChartTooltip = ({ x, data, ...rest }: { x: number; data: any }) => {
+  return (
+    <Crosshair {...rest} values={[{ x, y: 20 }]}>
+      <div style={{ background: '#555', color: 'white', padding: '.5em' }}>
+        <div>Build:&nbsp;{x}</div>
+        <div>Date:&nbsp;{data.ts}</div>
+        <table className="tooltip">
+          <thead>
+            <tr>
+              <th>Example</th>
+              <th>Min</th>
+              <th>Median</th>
+              <th>Max</th>
+            </tr>
+          </thead>
+          <tbody>
+            {Object.keys(data.performance)
+              .sort()
+              .map(chartName => (
+                <tr key={chartName}>
+                  <td>{chartName}</td>
+                  <td>{_.get(data, `performance.${chartName}.actualTime.min`, '-')}</td>
+                  <td>{_.get(data, `performance.${chartName}.actualTime.median`, '-')}</td>
+                  <td>{_.get(data, `performance.${chartName}.actualTime.max`, '-')}</td>
+                </tr>
+              ))}
+          </tbody>
+        </table>
+      </div>
+    </Crosshair>
+  )
+}
+
+/**
+ * Draws a performance chart for all items in perfData.performance.
+ * Shows tooltip with details for selected build on mouse hover.
+ * x-axis is a build number
+ * y-axis is a render time
+ */
+const PerfChart: React.FC<any> = ({ perfData }) => {
+  const availableCharts: string[] = perfData
+    .reduce(
+      (acc, next) => {
+        return Array.from(new Set([...acc, ...Object.keys(next.performance)]))
+      },
+      [] as string[],
+    )
+    .sort()
```

Comment:
What it does? Can we normalize data in `Provider` that fetches data?

Reply:
This creates a list of all charts which have any entry in any of the samples (=all available charts). I think we should not normalize the data anywhere else because the structure which we currently have is useful when walking through the data.